WordPress/Classes/ViewRelated/Blog/QuickStartTours.swift

```diff
@@ -171,6 +171,66 @@ struct QuickStartFollowTour: QuickStartTour {
     }
 }
 
+struct QuickStartSiteIconTour: QuickStartTour {
+    let key = "quick-start-site-icon-tour"
+    let analyticsKey = "site_icon"
+    let title = NSLocalizedString("Upload a site icon", comment: "Title of a Quick Start Tour")
+    let description = NSLocalizedString("Your visitors will see your icon in their browser. Add a custom icon for a polished, pro look.", comment: "Description of a Quick Start Tour")
+    let icon = Gridicon.iconOfType(.globe)
+    let suggestionNoText = Strings.notNow
+    let suggestionYesText = Strings.yesShowMe
+
+    var waypoints: [WayPoint] = []
+}
+
+struct QuickStartNewPageTour: QuickStartTour {
+    let key = "quick-start-new-page-tour"
+    let analyticsKey = "new_page"
+    let title = NSLocalizedString("Create a new page", comment: "Title of a Quick Start Tour")
+    let description = NSLocalizedString("Add a page for key content — an “About” page is a great start.", comment: "Description of a Quick Start Tour")
+    let icon = Gridicon.iconOfType(.pages)
+    let suggestionNoText = Strings.notNow
+    let suggestionYesText = Strings.yesShowMe
+
+    var waypoints: [WayPoint] = []
+}
+
+struct QuickStartPostSharingTour: QuickStartTour {
+    let key = "quick-start-post-sharing-tour"
+    let analyticsKey = "post_sharing"
+    let title = NSLocalizedString("Enable post sharing", comment: "Title of a Quick Start Tour")
+    let description = NSLocalizedString("Automatically share new posts to your social media accounts.", comment: "Description of a Quick Start Tour")
+    let icon = Gridicon.iconOfType(.share)
+    let suggestionNoText = Strings.notNow
+    let suggestionYesText = Strings.yesShowMe
+
+    var waypoints: [WayPoint] = []
+}
+
+struct QuickStartCheckStatsTour: QuickStartTour {
+    let key = "quick-start-check-stats-tour"
+    let analyticsKey = "check_stats"
+    let title = NSLocalizedString("Check your site stats", comment: "Title of a Quick Start Tour")
+    let description = NSLocalizedString("Keep up to date on your site’s performance.", comment: "Description of a Quick Start Tour")
+    let icon = Gridicon.iconOfType(.statsAlt)
+    let suggestionNoText = Strings.notNow
+    let suggestionYesText = Strings.yesShowMe
+
+    var waypoints: [WayPoint] = []
+}
+
+struct QuickStartExplorePlansTour: QuickStartTour {
+    let key = "quick-start-explore-plans-tour"
+    let analyticsKey = "explore_plans"
+    let title = NSLocalizedString("Explore plans", comment: "Title of a Quick Start Tour")
+    let description = NSLocalizedString("Learn about the marketing and SEO tools in our paid plans.", comment: "Description of a Quick Start Tour")
+    let icon = Gridicon.iconOfType(.plans)
+    let suggestionNoText = Strings.notNow
+    let suggestionYesText = Strings.yesShowMe
+
+    var waypoints: [WayPoint] = []
+}
+
```

Comment:
Instead of implementing `waypoints` every time to return an empty array, what do you think if we extend `QuickStartTour`?
Like:
```
extension QuickStartTour {
    var waypoints: [WayPoint] = []
}
```

Reply:
I'd love that, but extensions can't have stored properties. If you can think of another way to do it, I'm all ears 😀

Reply:
Oh yes, my bad! I meant a computed property when I wrote that 🤦‍♂️

Reply:
That seems to work. Will have to remember to remove it once the waypoints actually exist for each tour, unless we figure it's still useful. 👍